lab-isaac/model/toy.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const debug = require('debug')('http:model-toy');
+const mongoose = require('mongoose');
+
+const Toy = mongoose.Schema({
+  name: {type: String, require: true},
```

Comment:
I was having trouble making requests that had incorrect data and they were succeeding. I think you need to change 'require' to required'.